src/format/reveal/format-reveal.ts

```diff
@@ -223,6 +224,13 @@ export function revealjsFormat() {
           metadataOverride.previewLinks = false;
         }
 
+        // specify auto-stretch if there is no boolean 'auto-stretch'
+        const autoStretch =
+          typeof (format.metadata[kAutoStretch]) !== "boolean";
+        if (autoStretch) {
+          metadataOverride[kAutoStretch] = true;
+        }
```

Comment:
I change the way the default `auto-stretch` value is done to do it like `controlsAuto ` so that it is `true` by default, even in the case `auto-stretch` is set to a non boolean. Previously, if user set to non boolean, there was no check of the value

Is that ok ?

Reply:
I don't think we need to do this here (that's there because `controls` is also a valid reveal setting that only takes a boolean so we need the sideband `controlsAuto`.

I think it's preferable to do it the way you had it before (note that we are going to validate the schema so they'll actually get a runtime error if they set it to a non-boolean)

Reply:
> (note that we are going to validate the schema so they'll actually get a runtime error if they set it to a non-boolean)

Oh that is the part I did not get - I found the file. Do I need to something in `\src\resources\schema\format-metadata-reveal.yml` now ? Or this will be for later ?

Reply:
That work has continued on the `feature/yaml-validation` branch so you can add the entry to that branch (or I have a note to take care of this myself as I am frequently on that branch)